compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -197,7 +213,11 @@ cmdLint numProcessors =
   where
     cmd = execLint
         <$> many inputFileOpt
-        <*> optionsParser numProcessors (EnableScenarioService False) optPackageName
+        <*> optionsParser
+              numProcessors
+              (EnableScenarioService False)
+              optPackageName
+              enabledDlintUsageParser
```

Comment:
`cmdLint` only allows the user to change the configuration of the linter, but not to turn it off. What would be the point of `daml damlc lint --without-dlint`?